scripts/misc/generate_vep_examples.pl

```diff
@@ -142,6 +144,63 @@ =head1 CONTACT
     $b->length <=> $a->length} @{$sa->fetch_all('toplevel')
   };
   
+  # special address of bacteria species due to:
+  # a) undef seq_region_start/end issue unless explicitely defined in variationFeature and
+  # b) VariationFeature with seq_region_start/end defined for non-bacteria species returning differnet results than with them left out
+  # c) transcriptVariation consequence apprearing intergenic when missense expected possibly due to real/fake adaptors mix
+  if ($div_bacteria) {
+    my ($slice, $trs);
+
+    # create a missense
+    my $tr = undef;
+    while(!defined($tr)) {
+      next SPECIES unless scalar @slices;
+      ($slice, $trs) = @{select_slice(\@slices)};
+      $tr = select_transcript($trs, $div_bacteria);
+    }
+    my $pos = $tr->coding_region_start + 3;
+    my $sub_slice = $slice->sub_Slice($pos, $pos);
+    my $ref_seq = $sub_slice->seq;
+
+    my @tmp_alts = sort {rand() <=> rand()} grep {$_ ne $ref_seq} @alts;
+    my $alt = shift @tmp_alts;
+
+    #VariationFeature objects for bacteria have to be created explicitly specifying seq_region_start,seq_region_end, this will prevent these two being undef in subsequent uses, this is known in eg bacteria
+    my $tmp_vf = Bio::EnsEMBL::Variation::VariationFeature->new_fast({
+      start          => $pos,
+      end            => $pos,
+      allele_string  => $ref_seq.'/'.$alt,
+      strand         => 1,
+      map_weight     => 1,
+      adaptor        => $vfa,
+      slice          => $slice,
+      seq_region_start => $slice->seq_region_start,
+      seq_region_end   => $slice->seq_region_end
+    });
+    dump_vf($tmp_vf, \%files, \%web_data);
+
+    # create a frameshift
+    $tr = undef;
+    while(!defined($tr)) {
+      $tr = select_transcript($trs,$div_bacteria);
+    }
+    $pos = $tr->coding_region_start + 3;
```

Comment:
Might be misleading to use the same variable name `$pos` with different values.
And it would be the same for `$tmp_vf` and `$ref_seq`.